lib/systems/include/PedalsSystem.h

```diff
@@ -153,19 +122,28 @@ class PedalsSystem
     /// @param max
```

Comment:
Please complete this documentation header. Min/max relative to what? What does implaus_margin_scale mean? What ranges are valid for implaus_margin_scale? *please specify*.